spec/hydra/works/services/generic_file/persist_derivative_spec.rb

```diff
@@ -1,21 +0,0 @@
-require 'spec_helper'
-
-describe Hydra::Works::PersistDerivative do
-
-  let(:generic_file)        { Hydra::Works::GenericFile::Base.new }
-  let(:filename)            { "sample-file.pdf" }
-  let(:file)                { File.open(File.join(fixture_path, filename)) }
-  let(:type)                { ::RDF::URI("http://pcdm.org/use#ExtractedText") }
-  let(:service_type)        { ::RDF::URI("http://pcdm.org/use#ServiceFile") }
-
-  it "persists a service file by default" do
```

Comment:
Can you explain why these tests are going away? Are they already covered by more specific tests below? The level of abstraction seems to be different in this spec.